app/javascript/dashboard/components-next/NewConversation/ComposeConversation.vue

```diff
@@ -243,4 +262,45 @@ useKeyboardEvents(keyboardEvents);
       @discard="closeCompose"
     />
   </div>
+
+  <template v-else>
```

Comment:
Let's not do this, this makes the component unnecessarily complicated. We can manage the two different UIs in the same component above. As I see the only change is the extra div, We can add it as a `<template>` tag and render it as a div when required.

Reply:
@scmmishra Done https://github.com/chatwoot/chatwoot/pull/11085/commits/f57fabcc4a325394e3d1c23472dff182ad4208cd